stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/list/CustomEventHandler.kt

```diff
@@ -82,6 +83,21 @@ class CustomChatEventHandler(private val channels: StateFlow<List<Channel>>) : B
         }
     }
 
+    private fun handleMemberRemoval(
```

Comment:
This class is only used in the Sample app. Shouldn't we add this same logic in `DefaultChatEventHandler`